### PR TITLE
[TIDOC-3126] Remove add/remove API's from non-views

### PR DESCRIPTION
**Pull request by hansemannn**
**JIRA:** https://jira.appcelerator.org/browse/TIDOC-3126

### CODE REVIEW

**Comment by hansemannn** [comment]
@Topener Can you review this as well?

File: apidoc/Titanium/UI/Picker.yml [hunk 1]
```diff
@@ -39,11 +39,10 @@ description: |
 extends: Titanium.UI.View
 excludes:
     properties: [
-        anchorPoint,backgroundDisabledColor,backgroundDisabledImage,backgroundFocusedColor,
+        anchorPoint,backgroundDisabledColor,backgroundDisabledImage,backgroundFocusedColor,children
         backgroundFocusedImage,backgroundGradient,backgroundImage,backgroundLeftCap,backgroundRepeat,
         backgroundSelectedColor,backgroundSelectedImage,backgroundTopCap,center,enabled,layout,tintColor,zIndex]
-    methods: [ remove, removeAllChildren ]
-
+    methods: [add, remove, removeAllChildren, replaceAt]
```
Comment (Topener) [inline apidoc/Titanium/UI/Picker.yml]: According to the docs themselves, you add data to the picker using the `add` method... don't think it should be removed

Comment (hansemannn) [reply apidoc/Titanium/UI/Picker.yml]: Good catch, updated!

File: apidoc/Titanium/UI/iOS/Toolbar.yml [hunk 2]
```diff
@@ -33,12 +33,15 @@ description: |
     *  [Toolbar in the iOS Human Interface Guidelines.](https://developer.apple.com/library/ios/documentation/userexperience/conceptual/mobilehig/Bars.html#//apple_ref/doc/uid/TP40006556-CH12-SW4)
 
 extends: Titanium.UI.View
-excludes: { properties: [ backgroundColor, backgroundDisabledColor, backgroundDisabledImage,         
+excludes:
+    properties: [backgroundColor, backgroundDisabledColor, backgroundDisabledImage,         
             backgroundFocusedColor, backgroundFocusedImage,
             backgroundLeftCap, backgroundTopCap, backgroundRepeat,
             backgroundSelectedColor, backgroundSelectedImage, layout,
-            softKeyboardOnFocus ],
```
Comment (jawa9000) [inline apidoc/Titanium/UI/iOS/Toolbar.yml]: I think we need to drop the comma on this line.

**Review by jawa9000 — DISMISSED** [review]
Left one comment. Otherwise, it looks good to me pending any linting tests.

File: apidoc/Titanium/UI/iOS/Toolbar.yml (hunk 2 above)
Comment (hansemannn) [reply apidoc/Titanium/UI/iOS/Toolbar.yml]: It is a removed line, the new line is 41 (`softKeyboardOnFocus]`)

**Review by Topener — APPROVED** [review]

**Review by sgtcoolguy — COMMENTED** [review]
Should double-check with Kota about some of these since I think you can add/remove children on some of these UI elements there.

File: apidoc/Titanium/UI/Button.yml [hunk 3]
```diff
@@ -133,25 +133,10 @@ description: |
     when the button is pressed.
 
 extends: Titanium.UI.View
+excludes:
+    properties: [children]
+    methods: [add, remove, removeAllChildren, replaceAt]
 since: "0.8"
-methods:
```
[the remark this answers is not in the document]
Comment (infosia) [reply apidoc/Titanium/UI/Button.yml]: No, adding children to Buttons does not work on Windows.

File: apidoc/Titanium/UI/Label.yml [hunk 4]
```diff
@@ -4,24 +4,9 @@ summary: A text label, with optional background image.
 description: Use the <Titanium.UI.createLabel> method or **`<Label>`** Alloy element to create a label.
 extends: Titanium.UI.View
 since: "0.8"
-
-methods:
-    # main doc inherited from Ti.UI.View.add
-  - name: add
```
[the remark this answers is not in the document]
Comment (infosia) [reply apidoc/Titanium/UI/Label.yml]: Adding children to Labels does not work on Windows.

**Comment by sgtcoolguy** [comment]
Manually fixed conflicts, rebased into single commit and merged to master.

**Pull request closed without merging**